src/Control/Effect/Cull.hs

```diff
@@ -1,4 +1,12 @@
 {-# LANGUAGE DeriveFunctor, ExistentialQuantification, FlexibleContexts, StandaloneDeriving #-}
+{- | Provides an effect to cull choices in a given nondeterministic context. This effect is used in concert with 'Control.Effect.Choose.Choose' or 'Control.Effect.NonDet.NonDet'.
```

Comment:
```suggestion
{- | Provides an effect to cull choices in a given nondeterministic context. This effect is used in concert with 'Control.Effect.NonDet.NonDet'.
```

We only actually provide a carrier for `Cull` + `NonDet`, not `Cull` + `Choose`.